aicspylibczi/CziFile.py

```diff
@@ -285,9 +286,17 @@ def read_image(self, m_index: int = -1, **kwargs):
             sure the numpy.ndarray is interpretable. An example of the list is
                     [('S', 1), ('T', 1), ('C', 2), ('Z', 25), ('Y', 1024), ('X', 1024)]
                  so if you probed the numpy.ndarray with .shape you would get (1, 1, 2, 25, 1024, 1024).
+
         """
```

Comment:
Something about these docstring quotes seems off... It seems like there are three set's of them?